scripts/templates/print.hpp.mako

```diff
@@ -117,49 +121,49 @@ template <> struct is_handle<${th.make_type_name(n, tags, obj)}> : std::true_typ
 %endfor
 template <typename T>
 inline constexpr bool is_handle_v = is_handle<T>::value;
-template <typename T> inline void serializePtr(std::ostream &os, const T *ptr);
-template <typename T> inline void serializeFlag(std::ostream &os, uint32_t flag);
-template <typename T> inline void serializeTagged(std::ostream &os, const void *ptr, T value, size_t size);
+template <typename T> UR_DLLEXPORT inline ${x}_result_t UR_APICALL ${x}PtrPrint(std::ostream &os, const T *ptr);
```

Comment:
Since this is a `print` header, my preference would be to name all these `urPrintSomething`. Makes it easier to find all print functions.

Reply:
I would prefer to call these functions this way, too. I just adhered to the function naming convention of UR API (https://github.com/oneapi-src/unified-runtime/blob/main/scripts/core/CONTRIB.rst#naming-convention). Can I ignore this convention in the case of a printing API then?

Reply:
Renamed

Reply:
But we now have something like `ur_print::urFunctionParamsPrint`, which seems a bit excessive. Why not just `ur::printFunctionParams()` (i.e. rename namespace to `ur`, remove `ur` prefix from the function name and possibly start the function name with `print`)

Reply:
`ur::print::functionParams()` or maybe `ur::extras::printFunctionParams()`?

I think I'd prefer this whole thing be in its own namespace. Hypothetically, we might want to add some more C++ functionality or bindings, and imho it should be possible to use that functionality without having to import all these operator overloads.

Reply:
Well, you don't have to include ur_print.hpp header if you don't want to. But I like the `ur::extras::printFunctionParams()` or perhaps (`ur::info::printFunctionParams()`)